### DISCUSSION TITLE
toggle sidebar is under the sidebar so i cant close it

### DISCUSSION THREAD

**Discussion by Crowzed** [Laravel-Backpack/community-forum#1031] [Q&A (Help)]
![image](https://github.com/Laravel-Backpack/community-forum/assets/110807022/2d037d0f-4aab-43d9-a34f-8059e2b47fff)

**Comment by jcastroa87** [comment]
Hello @Crowzed 

Please share more information. What versions and packages are you using? What theme did you enable? Do you customize the theme?

Show me the output of:
```
php artisan backpack:version
```

Cheers.

**Reply by Crowzed** [reply]
![image](https://github.com/Laravel-Backpack/community-forum/assets/110807022/2dc459f1-43cb-4065-a2bd-bde4a4287395)

i didnt customize the theme

**Reply by jcastroa87** [reply]
Hello @Crowzed 

I'm sorry to be late here, this still happens in new versions?

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
